### DISCUSSION TITLE
posts/crowdsec-traefik/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#232] [Comments]
# posts/crowdsec-traefik/

CrowdSec is a free, open-source and collaborative IPS. Analyze behaviors, respond to attacks & share signals across the community. With CrowdSec, you can set up your own intrusion detection system that parses logs, detects and blocks threats, and shares bad actors with the larger CrowdSec community. It works great with a reverse proxy like traefik to help keep hackers at bay. Could this be a viable alternative to fail2ban?

https://technotim.live/posts/crowdsec-traefik/

**Comment by CRK1918** [comment]
Hello, Techno Tim, after I added the entryPoints config and I start it, it always  say forbidden, so I put # disable for now

My entryPoints config: 

entryPoints:
  http:
    address: ":80"
    http:
      redirections:
        entryPoint:
          to: https
          scheme: https
      ##Add CrowdSec middleware to an 80 entrypoint
     middlewares:
        - crowdsec-bouncer@file
  https:
    address: ":443"
    ##Add CrowdSec middleware to an 443 entrypoint
    http:
      middlewares:
        - crowdsec-bouncer@file




I run "docker exec crowdsec cscli decisions list" is no action, not sure what I missed?

**Reply by CRK1918** [reply]
Ok, it work for me if I only add it to https entry points

**Comment by StewartPolsky** [comment]
Hello!

I love watching your videos and you have helped me to learn so much.
I would love if you could do an updated video / tutorial on how to implement CrowdSec with Traefik using the plugin since the Traefik bouncer repo https://github.com/fbonalair/traefik-crowdsec-bouncer has not been updated in 2 years.

Thanks!

**Reply by luinniuxx** [reply]
I think this is a better option; try this plugin: https://github.com/maxlerebourg/crowdsec-bouncer-traefik-plugin
